www/src/data/tags-docs.js

```diff
@@ -0,0 +1,98 @@
+/* 
+  this is a whitelist of tags that can be displayed 
+  on the blog, the first value being the tag name, 
+  and the second value a link to an existing doc on
+  the topic, adding a new entry to this Map makes it
+  a new possible tag for blog posts
+*/
+export const TAGS_AND_DOCS = new Map([
+  [`accessibility`, `/docs/making-your-site-accessible/`],
+  [`agencies`, `/docs/gatsby-for-agencies/`],
+  [`algolia`, `/docs/adding-search-with-algolia/`],
+  [`apis`, ``],
+  [`apps`, ``],
+  [`authentication`, `/docs/building-a-site-with-authentication/`],
+  [`aws`, `/docs/deploying-to-s3-cloudfront/#getting-started---aws-cli`],
```

Comment:
```suggestion
  [`aws`, `/docs/deploying-to-s3-cloudfront/`],
```
The paragraph referenced is at the beginning of the page, so I think it makes sense to link to the overall doc.